### PR TITLE
Add is_rotated.py

### PR DESCRIPTION
**Pull request by victore07**
I added an algorithm to strings. This checks if a string is a rotated version of another string and returns a bool. A few tests were added to test_strings.py as well.

### CODE REVIEW

**Comment by danghai** [comment]
@victore07 Thanks your PR. I am going to merge when you fix a minor

File: strings/is_rotated.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+is_rotated("hello", "llohe") returns True
+is_rotated("hello", "helol") returns False
+
+accepts two strings
+returns bool
+"""
+
+def is_rotated(s1, s2):
+    if len(s1) == len(s2):
+        return s2 in s1+s1
```
Comment (danghai) [inline strings/is_rotated.py]: It should be `s1 + s1`

File: tests/test_strings.py [hunk 2]
```diff
@@ -165,6 +166,21 @@ def test_is_palindrome_stack(self):
...
+    Arguments:
+        unittest {[type]} -- [description]
+    """
+
+    def test_is_rotated(self):
+        self.assertTrue(is_rotated("hello", "hello"))
+        self.assertTrue(is_rotated("hello", "llohe"))
+        self.assertFalse(is_rotated("hello", "helol"))
+        self.assertFalse(is_rotated("hello", "lloh"))
+
```
Comment (danghai) [inline tests/test_strings.py]: Could you add 1 test case for empty string?

**Comment by danghai** [comment]
Thanks
